### DISCUSSION TITLE
Error executing an input file: Failed to get real path

### DISCUSSION THREAD

**Discussion by rl3fz** [idaholab/moose#18034] [Q&A Getting Started] [answered]
Hello everyone, I'm on Step 2 of the tutorial, and after following the instructions to execute the 'pressure_diffusion.i' file, I keep getting:

*** ERROR ***
Failed to get real path for pressure_diffusion.i

I was previously able to execute the file using the command `../babbler-opt -i pressure_diffusion.i` but not anymore. I've tried retracing my steps and updating MOOSE and Conda, but I'm not sure what changed.

Any advice would be greatly appreciated.

**Comment by GiudGiud** [comment, marked as answer]
Are you on Windows WSL?

**Reply by GiudGiud** [reply]
If so, see this thread https://github.com/idaholab/moose/discussions/18011

**Reply by GiudGiud** [reply]
If not, could you please let us know what `ls -l` returns in your directory. Thanks!

**Reply by GiudGiud** [reply]
Oh that other thread was with you too. Ok see right above

**Reply by rl3fz** [reply]
Yes, hello again! Contents and permissions:
-rw-r--r-- 1 rl3fz rl3fz  1550 Jun  8 11:55 pressure_diffusion.i.txt
-rw-r--r-- 1 rl3fz rl3fz 90744 Jun  8 11:55 pressure_diffusion_out.e

**Reply by GiudGiud** [reply]
ok so you renamed your file pressure_diffusion.i.txt
So there is no pressure_diffusion.i file in the folder. 
Re-rename it, or use the correct name after the `-i`

**Reply by rl3fz** [reply]
Ok, I understand now. The .i file came from a copy of the simple_diffusion input file, renamed and edited to match the code for pressure_diffusion. (I previously thought I was getting this error whether I included this file in the folder or not, but I must have just misspelled it at some point.) Is there anyway to create an input file from a txt file as the tutorial suggests? (The .txt is added automatically)

**Reply by permcody** [reply]
An input file is a text file (ASCII), the only difference is the extension. You should be able to rename the file (i.e. drop the .txt extension) without doing anything else to "create" your input file.

**Reply by rl3fz** [reply]
".txt" does not appear when I try to rename it; I can't edit it out manually. Is there a way to do so from the command line?

![image](https://user-images.githubusercontent.com/84426740/121419091-97adca00-c939-11eb-9c7a-2893f5aa9dc0.png)

**Reply by rl3fz** [reply]
(even when I end the name  with ".i")

**Reply by GiudGiud** [reply]
try `mv pressure_diffusion.i.txt pressure_diffusion.i`

**Reply by rl3fz** [reply]
This works, thank you!
